### PR TITLE
chore: nx migrations for nextjs

### PR DESCRIPTION
**Pull request by andresmgsl**

### CODE REVIEW

File: examples/znap-hats/app/web/app/actions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "name": "znap-action"
```
Comment (andresmgsl) [inline examples/znap-hats/app/web/app/actions.json:2]: just an example.. ill replace this with real data once we have it.

Comment (danmt) [reply examples/znap-hats/app/web/app/actions.json:2]: use this for now:

```json
{
  "rules": [
    {
      "pathPattern": "/**",
      "apiPath": "/api/**"
    }
  ]
}
```

File: examples/znap-hats/app/migrations.json
Comment (danmt) [inline examples/znap-hats/app/migrations.json:1]: migrations.json file is not meant to be committed.

**Review by danmt — CHANGES_REQUESTED** [review]
I dropped some fixed required

**Pull request merged**